src/main/java/ttt/LineGenerator.java

```diff
@@ -0,0 +1,55 @@
+package ttt;
+
+public class LineGenerator {
+    private final Cell[] grid;
+
+    public LineGenerator(Cell[] grid) {
+        this.grid = grid;
+    }
+
+    public Cell[][] linesForAllDirections() {
+        Cell[][] allLines = new Cell[8][];
+        allLines[0] = topRow();
+        allLines[1] = middleRow();
+        allLines[2] = bottomRow();
+        allLines[3] = leftColumn();
+        allLines[4] = middleColumn();
+        allLines[5] = rightColumn();
+        allLines[6] = backslashDiagonal();
+        allLines[7] = forwardslashDiagonal();
```

Comment:
I do not understand `backslashDiagonal` nor `fowardslashDiagonal`. I know what you are trying to convey, but the naming here is a little wonky...